docs/pg_bulkload-ja.html

```diff
@@ -774,12 +774,21 @@ <h3 id="rpm">RPM パッケージからのインストール</h3>
 <h2 id="requirement">動作環境</h2>
 <dl>
 <dt>PostgreSQLバージョン</dt>
-<dd>PostgreSQL 9.3, 9.4, 9.5, 9.6, 10, 11</dd>
+<dd>PostgreSQL 9.3, 9.4, 9.5, 9.6, 10, 11, 12</dd>
 <dt>OS</dt>
 <dd>RHEL 6/7</dd>
 </dl>
 
 <h2 id="releasenote">リリースノート</h2>
+
+<p>
+<h4>3.1.16</h4>
+<ul>
+<li>PostgreSQL 12に対応</li>
+<li>Fixed documentation to warn users of some risks of using parallel/multi-process mode</li>

```

Comment:
@MoonInsung please add the Japanese translation for this line and push it to this PR's branch.